src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -159,12 +160,16 @@ const CpsOnwardJourney = ({
   columnType,
   skipLink,
 }) => {
-  const a11yAttributes = {
-    as: 'section',
-    role: 'region',
-    'aria-labelledby': labelId,
-  };
   const { script, service, dir } = useContext(ServiceContext);
+  const { pageType } = useContext(RequestContext);
+
+  const isMostWatched = pageType === 'mostWatched';
+  const a11yAttributes = isMostWatched
+    ? {
+        as: 'div',
+      }
+    : { as: 'section', role: 'region', 'aria-labelledby': labelId };
```

Comment:
You might want to keep the label since it's used by e2es. You should check with @LilyL0u on that. Otherwise, it looks good to me

Reply:
It actually still has the e2e label, so all good there I think! 👍 

<img width="407" alt="Screenshot 2020-09-29 at 09 26 07" src="https://user-images.githubusercontent.com/9042672/94532657-0fb5e300-0236-11eb-8bbd-7be7e242bf08.png">

Reply:
I use 'data-e2e=most-watched-heading' in most watched component tests, as well as 'data-e2e=radio-schedule' in schedule tests. They are useful labels :)